Backend/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+  <link rel = stylesheet type = text/css href = "{{url_for('static', filename = 'style.css')}}">
+</head>
+  <body>
+    <h2>Welcome to Your Lean Fridge!</h2>
+    <div id = 'page-wrap'>
+    <p>Got a new item? Add it <a href = '{{url_for('newItem', stock = stock)}}'>here</a>
+      {% with messages = get_flashed_messages() %}
+      {% if messages %}
+      <!--To display all the flashed messages in an unordered list -->
+      <ul>
+        {% for message in messages %}
+        <li><strong>{{message}}</strong></li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+      {% endwith %}
+
+
+    {% for item in stock %}
+  <div class = 'pane'>
+    <p>id: {{item.id}}  </p>
+    <p>item: {{item.name}} </p>
+    <p>total: {{item.total}} </p>
+    <a href = '{{url_for('editItem', stock = stock, item_id = item.id)}}'>Edit</a>
```

Comment:
when i click on this link, it makes the URL like this:
`http://0.0.0.0:5000/stock/2/edit/?stock=%3Clean_fridge_setup.Stock+object+at+0x10f37d7d0%3E&stock=%3Clean_fridge_setup.Stock+object+at+0x10f37ddd0%3E&stock=%3Clean_fridge_setup.Stock+object+at+0x10f37d6d0%3E&stock=%3Clean_fridge_setup.Stock+object+at+0x10f37dfd0%3E&stock=%3Clean_fridge_setup.Stock+object+at+0x10f37dc90%3E`

the parts after the ? seem a bit strange. is this discussed in the course?